quarkus-build-caching-extension/pom.xml

```diff
@@ -143,6 +143,15 @@
                 </executions>
             </plugin>
         </plugins>
+        <resources>
+            <resource>
+                <targetPath>META-INF</targetPath>
```

Comment:
Will this include the license at `META-INF/LICENSE` in the JAR? We want it at the top level of the JAR.

Reply:
Yes it will include the `LICENSE` file in the `JAR` but in the `META-INF` folder.
Why do you want it at the top-level?

Reply:
Adjusted to root level for consistency